mobius-core/src/main/java/com/spotify/mobius/MobiusLoop.java

```diff
@@ -52,14 +52,19 @@
   private volatile boolean disposed;
 
   static <M, E, F> MobiusLoop<M, E, F> create(
-      MobiusStore<M, E, F> store,
+      Update<M, E, F> update,
+      M startModel,
+      Iterable<F> startEffects,
```

Comment:
Should this be `Iterable<F>` or `Set<F>`?

Reply:
I don't think that should matter because it's not a public method?

Reply:
that was my reasoning (plus Set wasn't already imported)